### DISCUSSION TITLE
Composer is unable to install mpdf/mpdf 8.1.0 on PHP8.2 or PHP8.3

### DISCUSSION THREAD

**Discussion by sameernaqvi1** [mpdf/mpdf#1999] [General]
Hi
I am using mpf/mpdf with PHP8.1 and it works well. Now our system requirement is to upgrade to php8.2 and in a 6-month time to php8.3
Right now I am facing the following issue when I try to install it via Composer.  I want to ask do you guys have any plans to upgrade it so it will support PHP8.2 or PHP8.3 in the near future. 

Error from Composer:
`- Root composer.json requires mpdf/mpdf 8.1.0 -> satisfiable by mpdf/mpdf[v8.1.0].
    - mpdf/mpdf v8.1.0 requires php ^5.6 || ^7.0 || ~8.0.0 || ~8.1.0 -> your php version (8.2.16) does not satisfy that requirement.`

**Comment by finwe** [comment]
PHP 8.2 is supported since mPDF 8.1.3, update your composer constraint. Right in the Readme: https://github.com/mpdf/mpdf

**Reply by sameernaqvi1** [reply]
Apologies I overlooked the Readme

I saw the releases section and it still shows that **8.1.0** is the latest version.


![Screenshot 2024-03-11 at 12 24 28](https://github.com/mpdf/mpdf/assets/25508620/45bdb170-416f-4080-904a-81a5f5bac775)

**Reply by sameernaqvi1** [reply]
Thanks for your prompt help!

**Reply by finwe** [reply]
Announced releases at github are not the only releases. See https://packagist.org/packages/mpdf/mpdf
